src/rpc/blockchain.cpp

```diff
@@ -2615,6 +2615,9 @@ static RPCHelpMan dumptxoutset()
         "Write the serialized UTXO set to a file.",
         {
             {"path", RPCArg::Type::STR, RPCArg::Optional::NO, "Path to the output file. If relative, will be prefixed by datadir."},
+            {"height", RPCArg::Type::NUM, RPCArg::DefaultHint{"the current tip"},
```

Comment:
Right, I think that makes sense and this will be called mostly with this argument than anything else. That hadn't occurred to me so far honestly because I mostly wanted this for us to create snapshots that will potentially be added to the code in the future.

I have changed this to default to the latest valid snapshot height. That makes getting the tip more cumbersome for the users but I think that's ok since this should not be what most of the want, I think.